client/modules/IDE/components/Header/Toolbar.jsx

```diff
@@ -50,7 +45,6 @@ const Toolbar = (props) => {
         className="toolbar__play-sketch-button"
         onClick={() => {
           props.syncFileContent();
-          dispatch(startAccessibleSketch());
```

Comment:
We need to continue starting the sketch here.  It's super confusing but this button on lines 49-61 is a hidden version that's shown on screen readers only. Lines 62-73 are the button that you see.  So removing this completely will prevent the sketch from playing on screen readers.